backend/src/rtp.rs

```diff
@@ -1541,6 +1554,48 @@ pub struct Endpoint {
     rtx_sender: RtxSender,
 }
 
+#[derive(thiserror::Error, Debug, Eq, PartialEq)]
+pub enum EndpointError {
+    #[error("SRTP decryption error: {0}")]
+    SrtpDecryptionError(String),
+
+    #[error("SRTP encryption error: {0}")]
+    SrtpEncryptionError(String),
+
+    #[error("RTCP sender SSRC not set")]
+    RtcpSenderSsrcNotSet,
+
+    #[error("Invalid outgoing SRTCP index")]
+    InvalidOutgoingSrtcpIndex,
+
+    #[error("Incoming SSRC state not found for SSRC: {0}")]
+    IncomingSsrcStateNotFound(Ssrc),
+
+    #[error("Transport-CC receiver error: {0}")]
+    TccReceiverError(String),
+
+    #[error("Transport-CC sender error: {0}")]
+    TccSenderError(String),
+
+    #[error("Max received TCC sequence number error: {0}")]
+    MaxReceivedTccSeqnumError(String),
+
+    #[error("RTX sender error: {0}")]
+    RtxSenderError(String),
```

Comment:
I don't see this being used. Generally, we want to avoid things that are not going to be used. It just increases the maintenance burden.

Reply:
I will purge the unnesecary variants.